interpreter/cling/tools/packaging/cpt.py

```diff
@@ -276,18 +278,15 @@ def download_llvm_binary():
         if DIST == "Ubuntu" and REV == '16.04' and is_os_64bit():
             download_link = 'http://releases.llvm.org/5.0.2/clang+llvm-5.0.2-x86_64-linux-gnu-ubuntu-16.04.tar.xz'
             exec_subprocess_call('wget %s' % download_link, workdir)
-            exec_subprocess_call('tar xvf clang+llvm-5.0.2-x86_64-linux-gnu-ubuntu-16.04.tar.xz', workdir)
-            exec_subprocess_call('mv clang+llvm-5.0.2-x86_64-linux-gnu-ubuntu-16.04 %s' % srcdir, workdir)
+            extract_tar(workdir, 'clang+llvm-5.0.2-x86_64-linux-gnu-ubuntu-16.04.tar.xz')
```

Comment:
Did you test this?

According to the `mv` in line 280, shouldn't this be uncompressed in `srcdir` instead of `workdir`?  (Same for the other two occurrences below).

Reply:
Yeah you’re right I’ll change that thank you

Reply:
In general I think that we should add an option `--test` to cpt where we will write a bunch of integration tests, testing each option. Otherwise we always risk of leaving something untested. @saisoma123 can you work on this?

Reply:
Yeah I'll get to work on that

Reply:
@vgvassilev Fixed up the code as well

Reply:
@jalopezg-r00t, is that good to go?

Reply:
Yeah just tested it, it extracted to the right path.